source/js/Directions.js

```diff
@@ -20,31 +124,114 @@ class Directions extends HTMLElement {
     const directionList = searchForKey(data, "recipeInstructions");
 
     const wrapper = document.createElement("article");
-    wrapper.classList.add("example");
+    wrapper.classList.add("background");
 
     const directions = document.createElement("p");
     directions.classList.add("directions");
     directions.textContent = "Directions";
+    directions.classList.add("header");
     wrapper.appendChild(directions);
 
+    // let button_wrapper = document.createElement('button_wrapper');
+    // button_wrapper.classList.add('buttonstyle');
+
     // Parsing data to create the direction list.
     const list = document.createElement("ol");
+    list.classList.add("olStyle");
     for (let i = 0; i < directionList.length; i++) {
-      const listItem = document.createElement("li");
-      listItem.textContent = `${directionList[i].name}`;
-      list.appendChild(listItem);
-
+      let direction = document.createElement("div");
+      let x = i+1;
+      let text_button = document.createElement('button');
+      let wrapper = document.createElement("div");
+      text_button.appendChild(wrapper);
+      let cb = document.createElement("input");
```

Comment:
Codacy has a fix for the issue: ['cb' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
      const cb = document.createElement("input");
```